README.md

```diff
@@ -164,48 +164,109 @@ Example of configuring test task for build type `debug` in Android:
 <details open>
 <summary>Kotlin</summary>
 
+`build.gradle.kts` (Project)
```

Comment:
This example is located in the section `Configuration / Configuring JVM test task`, I think it's worth creating a new section `Configuring Android application`, and leave this code as it is.

Reply:
Done